_data/conference.yml

```
@@ -211,34 +212,61 @@ subscribe-title: Join our conference
 subscribe-subtitle: Subscribe
 subscribe-description: Subscribe and follow <a href="https://bsky.app/profile/scaladays.org" target="_blank" rel="noopener noreferrer">@ScalaDays</a> on BlueSky for the latest conference updates.
 
+
 # The venue
-# ----------------------------------------------------------------------------------------------------------
+# -----------------------------------------------------------------------------------------------------------
 # ----------------------------------------------------------------------------------------------------------
 
 the-venue:
   - icon: img/assets/icon-about-location.svg
     title: Location
-    description: Scala Days Madrid is located around the Chamartín train station, in the northern part of Madrid's city center. The address is Calle de Agustín de Foxá, 40, 28036 Madrid.
+    description: Scala Days Lausanne is located at the SwissTech Convention Center (STCC), situated on the EPFL campus in the western part of Lausanne. The address is Route Louis-Favre 2, 1024 Ecublens.
   - icon: img/assets/icon-airport.svg
     title: The Airport
-    description: Adolfo Suárez Madrid–Barajas Airport (MAD), or more commonly referred to as Madrid–Barajas Airport, is Spain’s largest airport and Europe’s second-largest behind Paris–Charles de Gaulle. Madrid-Barajas has four passenger terminals and is 12 kilometers from Madrid’s city center.
+    description: Geneva Airport (GVA), also known as Genève-Cointrin Airport, is Switzerland’s second-busiest airport behind Zurich. GVA offers two passenger terminals and is located approximately 4 kilometers from Geneva’s city center and around 60 kilometers from Lausanne.
```

Zürich should be written with an umlaut. 

```suggestion
    description: Geneva Airport (GVA), also known as Genève-Cointrin Airport, is Switzerland’s second-busiest airport behind Zürich. GVA offers two passenger terminals and is located approximately 4 kilometers from Geneva’s city center and around 60 kilometers from Lausanne.
```

---

I believe mentioning ZRH (Zürich Flughafen) and BSL (Basel-Mulhouse EuroAirport) is important.

Pushing it a bit but MXP (Milano Malpensa) + EC from Milano Centrale?

---

I agree with mentioning ZRH and BSL but the correct way of writing _"Zurich"_ in English is without umlaut (c.f. https://en.wiktionary.org/wiki/Zurich)

---

I added descriptions of ZRH and BSL/MLH/EAP